AnalysisStep/src/LHEHandler.cc

```diff
@@ -78,6 +81,11 @@ float LHEHandler::getLHEWeigh_AsMZUpDn(int whichUpDn, float defaultValue) const{
   else return defaultValue;
 }
 float const& LHEHandler::getPDFScale() const{ return PDFScale; }
+float LHEHandler::reweightNNLOtoNLO() const{
+  if (year == 2017)
+    return defaultNLOweight / getLHEWeight(0, 1);
+  throw cms::Exception("LHEWeights") << "Shouldn't be calling this function for " << year;
+}
```

Comment:
I would suggest a conceptual change:
- Rename this function to getGlobalNormFactor
- return 1 instead of throw
Would that work?

Reply:
If you want to change it, go ahead.

Reply:
Alright, as you wish.